CityPuzzle/Login And SingUp/LoginPage.xaml.cs

```diff
@@ -25,20 +24,16 @@ protected override void OnAppearing()
             base.OnAppearing();
             Vartotojo_vardas.Text = "";
             Slaptazodis.Text = "";
-            while (App.FilePath == null)
-            {
-                Thread.Sleep(500);
-                Console.WriteLine("miegu");
-            }
-            try
+            try //pastebejau kad meta exeptiona jei failo neranda, todel try idejau
```

Comment:
When catching this error something should happen and catch block should not be empty

Reply:
I moved try to sql.